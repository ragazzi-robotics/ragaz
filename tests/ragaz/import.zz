from inner_module import CONST, func1 as fn1, innest_module_1 as mod1, custom_range
import inner_module.innest_module_2


def main():

    for i in custom_range(0, 5):
        print(i)

    print(CONST)
    print(mod1.CONST)
    print(inner_module.innest_module_2.CONST)

    print(fn1())
