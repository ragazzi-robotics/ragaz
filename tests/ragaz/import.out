0
1
2
3
4
constant0
constant1
constant2
function1
